Lib/urllib/parse.py

```diff
@@ -908,7 +909,14 @@ def urlencode(query, doseq=False, safe='', encoding=None, errors=None,
                         l.append(k + '=' + elt)
     return '&'.join(l)
 
-def to_bytes(url):
+
+def to_bytes(url=''):
```

Comment:
When I was researching the codebase to see what changes needed to be made for a deprecation warning, I found `dist` and `linux_distribution` in platform and test.platform, so I copied the tests from there.  That change altered the signature from the original to the internal version of the function.  I didn't understand why, so I applied a similar change and I probably shouldn't have.  I've removed it.